.gitignore

```diff
@@ -48,4 +48,6 @@ test.conf
 spark-warehouse
 *.flattened-pom.xml
 
-seatunnel-examples
\ No newline at end of file
+seatunnel-examples
+
+connectors
```

Comment:
why modify this?

Reply:
> why modify this?

Because the test not use docker, so we need copy the connector jars to a dir, I copy the connector jars in project_root/connectors, so connectors dir need ignore.

Reply:
I update the test case and don't use connectors dir. So I revert the gitignore file.